src/server/routes/groups.ts

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const controllers = require('../controllers.ts');
+const groups = express.Router();
+
+groups
+  .route('/')
+  .post(controllers.createGroup)
+
+module.exports = groups;
```

Comment:
Are there any get routes set up for the groups to get the due_date for the timer?

Reply:
currently when groups are created, we will have information of the designated "due date" stored in state, hence we shouldn't need to return the due date as we already have access. but lmk if you would like

Reply:
since I'm already writing the docs right now, will add a return due date feature just for extra coverage. this will be documented in the documentation so stay tuned

Reply:
wouldn't that due_date just be stored in the OG users state? How are the friends going to see the timer on their own app when making their order?

Reply:
the only data that is being stored on the group table is the due_date. the due date is stored when the group is created. 

Because we are generating a time based on the existing due date from the DB, we just need to access the due date from the current group. 

Which reminds me, I forgot to add a getGroup route lol. adding now

Reply:
Yeah I was basically asking for a get group route so I could get the due_date